firebase-firestore/src/main/java/com/google/firebase/firestore/MetadataChanges.java

```diff
@@ -17,7 +17,7 @@
 import com.google.firebase.annotations.PublicApi;
 
 /**
- * Indicates whether metadata-only changes (i.e. only {@code DocumentSnapshot.getMetadata()} or
+ * Indicates whether metadata-only changes (i.e.&nbsp;only {@code DocumentSnapshot.getMetadata()} or
```

Comment:
Was this intended?

Reply:
Yes. Without this, javadoc cuts the summary fragment after the "i.e." (Javadoc assumes the first period followed by a space ends the summary fragment.) There aren't any good workarounds; adding nbsp tricks javadoc into not thinking there's a space there.

But Rachel's suggestions elsewhere provide another possibility; just spell it out in english. Done.